conf/modules.config

```diff
@@ -25,7 +25,6 @@ process {
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
-        container = "community.wave.seqera.io/library/multiqc:1.22.1--4886de6095538010"
```

Comment:
Rather than removing this, why isn't this updated to the most recent container (`community.wave.seqera.io/library/multiqc:1.24.1--789bc3917c8666da`) if you updated the MultiQC version and module?

Reply:
I tested the proposed fix in gitpod and that seemed to work!

Reply:
Since the MultiQC module was updated in this PR and I thought it would be more "precise" if only in the test.config, I fully excluded the older line in modules.config

I just wanted a fix for this issue easily accessible in the pipeline itself, but understand it's not the proper way - removing it from both test.config and modules.config, adding a note in usage.